lib/remap-lang/grammar.pest

```diff
@@ -0,0 +1,80 @@
+// Root ------------------------------------------------------------------------
+
+program    = _{ SOI ~ NEWLINE* ~ (expression ~ (NEWLINE+ ~ expression)*)* ~ NEWLINE* ~ EOI }
```

Comment:
I haven't looked at the code yet. Does this imply that every expression clones its result `Value` to produce a return value, or is there a smarter reference happening to avoid that?

Reply:
Yes, I believe `SOI` and `EOI` are required, at least from all the documentation and examples I've seen.